app/controllers/system_messages_controller.rb

```diff
@@ -15,7 +17,7 @@ def create
     @system_message.user = current_user
 
     if @system_message.save
-      @system_message.send!
+      @system_message.send!(current_user)
```

Comment:
Since we store the user in the SystemMessage model instance, do we need to pass this?

Reply:
nope removed